tests/vmi_gpu_test.go

```diff
@@ -79,7 +81,10 @@ var _ = Describe("[Serial][sig-compute]GPU", Serial, func() {
 			vmi, apiErr := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(randomVMI)
 			Expect(apiErr).ToNot(HaveOccurred())
 
-			pod := libvmi.GetPodByVirtualMachineInstance(vmi, util.NamespaceTestDefault)
+			Eventually(func() error {
```

Comment:
I think it is preferable to wait for a phase/condition over using this retry.

Reply:
Done